internal/pkg/manifest/lb_web_svc.go

```diff
@@ -169,6 +169,14 @@ func (s *LoadBalancedWebService) Port() (port uint16, ok bool) {
 	return aws.Uint16Value(s.ImageConfig.Port), true
 }
 
+// ServiceConnectEnabled returns if ServiceConnect is enabled or not.
+func (s *LoadBalancedWebService) ServiceConnectEnabled() bool {
+	if s.Network.Connect.EnableServiceConnect != nil && !*s.Network.Connect.EnableServiceConnect {
+		return false
+	}
+	return true
```

Comment:
I think this is equivalent:
```suggestion
	return s.Network.Connect.EnableServiceConnect == nil || *s.Network.Connect.EnableServiceConnect
```